examples/dashboard.ipynb

```diff
@@ -117,16 +112,13 @@
     "        return shaded.opts(alpha=self.data_opacity, show_legend=False)\n",
     "\n",
     "    def viewable(self,**kwargs):\n",
-    "        data_dmap  = hv.DynamicMap(self.elem)\n",
-    "        rasterized = hv.util.Dynamic(data_dmap, operation=self.rasterize, streams=[hv.streams.RangeXY])\n",
-    "        \n",
-    "        c_stream   = hv.streams.Params(self, ['cmap', 'normalization'])\n",
-    "        s_stream   = hv.streams.Params(self, ['spreading'], rename={'spreading': 'px'})\n",
-    "        shaded     = spread(shade(rasterized, streams=[c_stream]), streams=[s_stream], how=\"add\")\n",
-    "        shaded     = hv.util.Dynamic(shaded, operation=self.apply_opacity)\n",
-    "        \n",
-    "        return hv.DynamicMap(self.tiles) * shaded * hv.DynamicMap(self.labels)\n",
+    "        rasterized = rasterize(hv.DynamicMap(self.elem), aggregator=self.aggregator, width=800, height=400)\n",
+    "        shaded     = shade(rasterized, cmap=self.param.cmap)\n",
+    "        spreaded   = spread(shaded, px=self.param.spreading, how=\"add\")\n",
+    "        dataplot   = spreaded.map(self.apply_opacity)\n",
```

Comment:
```suggestion
    "        dataplot   = spreaded.apply(self.apply_opacity)\n",
```

Reply:
Odd; I had already done that but it seems to have been overwritten somehow.